ansible_ai_connect/ai/api/serializers.py

```diff
@@ -308,6 +308,60 @@ class PlaybookExplanationFeedback(serializers.Serializer):
     )
 
 
+class ChatRequestSerializer(serializers.Serializer):
```

Comment:
are these just moved? or there are changes? if so, can you point them out, will be helpful to reviewers?

Reply:
just moved

Reply:
I just created mostly a new `ChatFeedback` serializer here, then reused the existing response from @TamiTakamiya , then moved this the request class as well to put all together.